presentation/src/main/java/com/android254/presentation/sessions/components/SessionsFilterPanel.kt

```diff
@@ -98,7 +98,7 @@ private fun loadFilters(): List<SessionsFilterOption> {
             type = SessionsFilterCategory.Level
         ),
         SessionsFilterOption(
-            label = "advanced",
+            label = "Advanced",
```

Comment:
Since you are here, do you mind extracting this to a string resource?

Reply:
On it!